cmd/zdb/zdb.c

```diff
@@ -3446,6 +3446,9 @@ dump_block_stats(spa_t *spa)
 		}
 	}
 
+	/* This is safe after zio_wait */
```

Comment:
Let's be even more explicit in this comment since the race here is subtle.  How about something like this.

```
/ * Done after zio_wait() since zcb_haderrors is modified in zdb_blkptr_done() */
```

Reply:
Will do.